Tasks/Liavonchyk/Task6/Task6.py

```diff
@@ -0,0 +1,38 @@
+def calc_array_sum(in_list):
+    sum = 0 #counter
+    def array_sum(in_list): #calc sum of level elements
```

Comment:
yes, but why exactly do you need this function nested inside **calc_array_sum**? be aware that this is not conventional recursion, while **array_sum** is

I suggest you to define one function inside any other ONLY in case of closure needed